packages/picasso/src/QuillEditor/utils/getEditorChangeHandler/test.ts

```diff
@@ -1,10 +1,9 @@
 import { act } from '@toptal/picasso/test-utils'
-import Quill from 'quill'
-import Delta from 'quill-delta'
+import Quill, { DeltaStatic } from 'quill'
 
 import getEditorChangeHandler from './getEditorChangeHandler'
 
-const deltaMock = new Delta().insert('foo')
+const deltaMock = { ops: [{ insert: 'foo' }] } as DeltaStatic
```

Comment:
I saw this `as DeltaStatic` a few times so would it make sense to provide our own class to make it look like the previous version?

```suggestion
const deltaMock = new OurDelta().insert('foo')
```